Cargo.lock

```diff
@@ -102,9 +102,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.97"
+version = "1.0.98"
```

Comment:
I think this file should be deleted. The module is a library, so it shouldn't have a checked in lockfile, right?